st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -170,8 +170,15 @@ def run(self, action_parameters):
                                                        **options)
 
         status = ACTIONEXEC_STATUS_RUNNING
-        query_context = {'mistral_execution_id': str(execution.id)}
-        LOG.info('Mistral query_context is %s' % query_context)
         partial_results = {'tasks': []}
 
-        return (status, partial_results, query_context)
+        context = {
```

Comment:
What would happen if you were only to expand the context to be structured and make no other change? Of course the corresponding change is necessary in  `st2actions/st2actions/query/mistral/v2.py` but nothing else - right?